clang/include/clang/Basic/LangOptions.def

```diff
@@ -186,6 +186,7 @@ COMPATIBLE_LANGOPT(RecoveryAST, 1, 1, "Preserve expressions in AST when encounte
 COMPATIBLE_LANGOPT(RecoveryASTType, 1, 1, "Preserve the type in recovery expressions")
 
 BENIGN_LANGOPT(ThreadsafeStatics , 1, 1, "thread-safe static initializers")
+BENIGN_LANGOPT(TlsGuards , 1, 1, "on-demand TLS initialization")
```

Comment:
Should this be a CODEGENOPT instead of a LANGOPT?  I guess it's not a big difference either way.  (ThreadsafeStatics is a LANGOPT because it also controls a preprocessor define.)

Reply:
done. moved to codegen options